packages/twenty-front/src/modules/action-menu/actions/record-actions/single-record/workflow-actions/hooks/useSeeWorkflowExecutionsWorkflowSingleRecordAction.ts

```diff
@@ -0,0 +1,66 @@
+import { useActionMenuEntries } from '@/action-menu/hooks/useActionMenuEntries';
+import {
+  ActionMenuEntryScope,
+  ActionMenuEntryType,
+} from '@/action-menu/types/ActionMenuEntry';
+import { CoreObjectNamePlural } from '@/object-metadata/types/CoreObjectNamePlural';
+import { FilterQueryParams } from '@/views/hooks/internal/useViewFromQueryParams';
+import { ViewFilterOperand } from '@/views/types/ViewFilterOperand';
+import { useWorkflowWithCurrentVersion } from '@/workflow/hooks/useWorkflowWithCurrentVersion';
+import qs from 'qs';
+import { useNavigate } from 'react-router-dom';
+import { IconHistoryToggle, isDefined } from 'twenty-ui';
+
+export const useSeeWorkflowExecutionsWorkflowSingleRecordAction = ({
+  workflowId,
+}: {
+  workflowId: string;
+}) => {
+  const { addActionMenuEntry, removeActionMenuEntry } = useActionMenuEntries();
+
+  const workflowWithCurrentVersion = useWorkflowWithCurrentVersion(workflowId);
+
+  const navigate = useNavigate();
+
+  const registerSeeWorkflowExecutionsWorkflowSingleRecordAction = ({
```

Comment:
For consistence, lets use runs instead of executions
```suggestion
  const registerSeeWorkflowRunsWorkflowSingleRecordAction = ({
```